### PR TITLE
Bug 1344274 - Add shutdownkill to main_summary

### PR DESCRIPTION
**Pull request by saptarshiguha**
Added support for shutdown_kill

1.  Added definitions to MainSummary.md
2. Added a test
3. Edited buildSchema

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala [hunk 1]
```diff
@@ -459,6 +459,8 @@ object MainSummaryView {
         hsum(keyedHistograms \ "PROCESS_CRASH_SUBMIT_SUCCESS" \ "main-crash"),
         hsum(keyedHistograms \ "PROCESS_CRASH_SUBMIT_SUCCESS" \ "content-crash"),
         hsum(keyedHistograms \ "PROCESS_CRASH_SUBMIT_SUCCESS" \ "plugin-crash"),
+        hsum(keyedHistograms \ "SUBPROCESS_KILL_HARD" \ "ShutDownKill"),
+
```
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala]: Please remove this blank line

**Comment by saptarshiguha** [comment]
Will do. I've filed a bug to keep it consisent with other changes e.g.
wow64.

**Comment by maurodoglio** [comment]
One last bit and we can merge it. Could you please remove the empty merge commit and use a rebase instead? 😺

**Comment by maurodoglio** [comment]
Closing this in favor of https://github.com/mozilla/telemetry-batch-view/pull/186

**Pull request closed without merging**